CMakeLists.txt

```diff
@@ -18,8 +18,6 @@ include(CheckCSourceCompiles)
 enable_testing()
 
 check_include_file(sys/types.h HAVE_SYS_TYPES_H)
-check_include_file(stdint.h    HAVE_STDINT_H)
```

Comment:
stdint.h is C99, right?

Reply:
As an aside, I wish we could already declare C89 as a dead dialect...  I'll direct my wishes to GCC, in the hope that they start considering deprecation of C89 :)